docs/core/project-sdk/msbuild-props.md

```diff
@@ -524,58 +524,147 @@ The `EnableDefaultNoneItems` property controls whether `None` items (files that
 The following MSBuild properties are documented in this section:
 
 - [AnalysisLevel](#analysislevel)
+- [AnalysisLevel\<Category>](#analysislevelcategory)
 - [AnalysisMode](#analysismode)
+- [AnalysisMode\<Category>](#analysismodecategory)
 - [CodeAnalysisTreatWarningsAsErrors](#codeanalysistreatwarningsaserrors)
 - [EnableNETAnalyzers](#enablenetanalyzers)
 - [EnforceCodeStyleInBuild](#enforcecodestyleinbuild)
 
 ### AnalysisLevel
 
-The `AnalysisLevel` property lets you specify a code-analysis level. For example, if you want access to preview code analyzers, set `AnalysisLevel` to `preview`.
+The `AnalysisLevel` property lets you specify a version of code analyzers. For example, if you want access to preview code analyzers, set `AnalysisLevel` to `preview`.
```

Comment:
while "code-analysis level" is not very clear to begin with I am not certain that "version of code analyzers" is better. The intent is that each .NET release (5, 6, etc) has a default set of analysis rules (and therefore a default analysis level). To change this is to change which set of analyzers run (i.e. all, none, some well-defined subset) but the _version_ of these analyzers does not change.